application/helpers/admin/template_helper.php

```diff
@@ -134,7 +134,11 @@ function templateExtractFilter($p_event, &$p_header)
     $aAllowExtensions[] = 'twig';
     $info = pathinfo($p_header['filename']);
 
-    if ($p_header['folder'] || !isset($info['extension']) || in_array($info['extension'], $aAllowExtensions)) {
+    if (isset($info['extension'])) {
+        $p_header['filename'] = preg_replace('/\.\w+$/', '.' . strtolower($info['extension']), $p_header['filename']);
```

Comment:
You rename it here ? 

I don't think it's a good idea : if user add <img src="DATA.PNG"> in is twig file

Reply:
> I don't think it's a good idea : if user add  in is twig file

He should edit it in that case.
Lower case extensions will minimize errors I believe.

@olleharstedt thoughts?

Reply:
>  Lower case extensions will minimize errors I believe.

There are no reason to didn't allow picture.PNG

Reply:
Done